tests/integration/test_feedback.py

```diff
@@ -69,8 +75,44 @@ def test_feedback_wrong_conversation_id():
     """Check if feedback with wrong conversation ID is not accepted by the service."""
     response = client.post(
         "/v1/feedback",
-        json={"conversation_id": 0, "feedback_object": "blah"},
+        json={"conversation_id": 0, "feedback_object": {"some": "data"}},
     )
     # for the request send w/o proper payload, the server
     # should respond with proper error code
     assert response.status_code == requests.codes.unprocessable_entity
+
+
+def test_feedback_list():
+    """Check if feedback list is returned."""
+    # store some feedback first
+    conversation_id = suid.get_suid()
+    response = client.post(
+        "/v1/feedback",
+        json={"conversation_id": conversation_id, "feedback_object": {"some": "data"}},
+    )
+    assert response.status_code == requests.codes.ok
+
+    response = client.get("/v1/feedback/list")
+
+    assert response.status_code == requests.codes.ok
+    assert response.json() == {"response": str([conversation_id])}
+
+
+def test_feedback_remove():
+    """Check if feedback is removed."""
+    # store some feedback first
+    conversation_id = suid.get_suid()
+    response = client.post(
+        "/v1/feedback",
+        json={"conversation_id": conversation_id, "feedback_object": {"some": "data"}},
+    )
+    assert response.status_code == requests.codes.ok
+
+    # The TestClient.delete() method from Starlette's TestClient does
+    # not accept a json argument - workaround it by using `requests`
```

Comment:
This suggests defining the endpoint in this manner `/v1/feedback/{feedback_id}`. As it is unsure that the endpoint will stay in place, I'm not sure if it makes sense to further invest in it. WDYT?

Reply:
i agree it's not worth investing in